src/components/screens/editor/editor-screen.tsx

```diff
@@ -97,6 +97,15 @@ const EditorScreen: React.FC = () => {
           data: { ...newMetaform } as any
         }
       });
+
+      await emailNotificationsApi.createEmailNotification({
+        metaformId: newMetaform.id!,
+        emailNotification: {
+          emails: [],
+          subjectTemplate: "Uusi vastaus sähköiselle lomakkeelle",
+          contentTemplate: "Järjestelmään on jätetty uusi vastaus sähköiselle lomakkeelle. <br/><br/> Voit käydä tarkastelemassa vastausta osoitteessa https://metaform-portal.etelasavonha.fi/admin"
```

Comment:
Could this be put into localizations?

Reply:
It should be static message, not localized.